<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMovie</class>
 <widget class="QDialog" name="AddMovie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSpinBox" name="addMovieYear">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>1980</number>
   </property>
   <property name="maximum">
    <number>2022</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addMovieDirector">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="addMoviePrice">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>12</number>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Lenght </string>
   </property>
  </widget>
  <widget class="QPushButton" name="addMovie">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add movie</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Director</string>
   </property>
  </widget>
  <widget class="QComboBox" name="addMovieCategory">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Horror</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Comedy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sci-Fi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Action</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Drama</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fantasy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Romance</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thriller</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Historical</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Musical</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addMovieTitle">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="addMovieLength">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>60</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
